docs/consume-packages/install-use-packages-visual-studio.md

```diff
@@ -14,155 +14,183 @@ f1_keywords:
 
-1. When managing packages for the solution, the UI lets you select the projects that are affected by the operations:
+1. In the **Manage NuGet Packages for Solution** window, select the projects that are affected by the operations.
 
-    ![Project selector when managing packages for the solution](media/SolutionPackagesUI.png)
+    ![Screenshot showing the Manage Packages for Solution window with multiple projects selected.](media/manage-packages-for-solution.png)
 
 ### Consolidate tab
 
-Developers typically consider it bad practice to use different versions of the same NuGet package across different projects in the same solution. When you choose to manage packages for a solution, the Package Manager UI provides a **Consolidate** tab on which you can easily see where packages with distinct version numbers are used by different projects in the solution:
+Developers typically consider it bad practice to use different versions of the same NuGet package across different projects in the same solution. Visual Studio allows you to use a common version for your NuGet packages. To do so, use the **Consolidate** tab of the **NuGet Package Manager** window to discover where packages with distinct version numbers are used by different projects in the solution.
+
+![Screenshot showing the Manage Packages for Solution window with the Consolidate tab selected.](media/consolidate-tab.png)
+
+In this example, the ClassLibrary1 project is using EntityFramework 6.2.0, whereas ConsoleApp1 is using EntityFramework 6.1.0. To consolidate package versions, follow these steps:
 
-![Package Manager UI Consolidate tab](media/ConsolidateTab.png)
+1. From the **Consolidate** tab, select the projects to update in the project list.
 
-In this example, the ClassLibrary1 project is using EntityFramework 6.2.0, whereas ConsoleApp1 is using EntityFramework 6.1.0. To consolidate package versions, do the following:
+1. Select the version to use for all these projects in the **Version** list.
 
-- Select the projects to update in the project list.
-- Select the version to use in all those projects in the **Version** control, such as EntityFramework 6.2.0.
-- Select the **Install** button.
+1. Select **Install**.
 
-The Package Manager installs the selected package version into all selected projects, after which the package no longer appears on the **Consolidate** tab.
+   The NuGet Package Manager installs the selected package version into all the selected projects, after which the package no longer appears on the **Consolidate** tab.
 
 ## Package sources
 
-To change the source from which Visual Studio obtains packages, select one from the source selector:
+To manage NuGet package sources, follow these steps:
 
-![Package source selector in the package manager UI](media/PackageSourceDropDown.png)
+1. To change the source from which Visual Studio obtains packages, select a source from the **Package source** selector.
```

Comment:
"Obtains packages" isn't quite accurate. The UI loads metadata from versions of a package based on the source selected. However, NuGet Restore doesn't care which drop-down source is selected.

> If a package isn't already installed, NuGet first attempts to retrieve it from the [cache](https://docs.microsoft.com/en-us/nuget/consume-packages/managing-the-global-packages-and-cache-folders). If the package isn't in the cache, NuGet tries to download the package from all enabled sources in the list at Tools > Options > NuGet Package Manager > Package Sources in Visual Studio.

https://docs.microsoft.com/en-us/nuget/consume-packages/package-restore#package-restore-overview
https://docs.microsoft.com/en-us/nuget/consume-packages/package-source-mapping

There is a new feature that *will do this*, called **Package Source Mapping**. See https://docs.microsoft.com/en-us/nuget/consume-packages/package-source-mapping